docs/settingUp.md

```diff
@@ -88,9 +88,14 @@ Eclipse IDE is our preferred development environment. Support requests related t
       ![devverification-guide-1.png](images/devverification-guide-1.png)
      
    * Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentations.
-     Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
-     * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
-     * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`
+   * Trailing whitespace: all forms of trailing whitespace is disallowed, blank lines included.
+   * Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
+     * Java:
+        * `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
+        * `Java → Editor → Save Actions → Additional actions → Configure → Remove trailing whitespace → All lines`
```

Comment:
I too think some instructions can be given at high-level and let the dev figure out how to do it.
I think dev needs to configure the IDE to help with removing white space in blank lines. Otherwise IDEs in fact add white space automatically to new lines in their effort to help maintain the same indentation level as the previous line.